### PR TITLE
Chores

### PR DESCRIPTION
**Pull request by Joseph-Burke**
## Chores (no ticket)

This PR contains 3 chore commits concerned with linting the codebase, updating the README, setting dependency versions.

These 3 commits are present in other branches such as `feat/sorting`, so the idea here is that we merge this into `develop` and reduce the diff between the branches, making subsequent merges plainer, quicker and safer.

I might recommend browsing through these commit-by-commit, as the commit messages provide some helpful context about what changes have been made :)

### CODE REVIEW

**Review by WillKocur — CHANGES_REQUESTED** [review]
Looks good - checked loading the site and both the list and api endpoints 👍 

One small comment about a strange whitespace thing and another about including instance methods, but happy for you to merge once addressed :)

File: wcmc_components/lib/wcmc_components/loadable.rb
Comment (Joseph-Burke) [reply wcmc_components/lib/wcmc_components/loadable.rb:8]: Not 100% sure _but_:

- The `base.send` line is from Ben's original code, 3 years old now.
- The `InstanceMethods` module (line 12) is empty.

Between those two facts, I think it's probably safe to remove this from Loadable as it has been from Filterable.

Comment (Joseph-Burke) [reply wcmc_components/lib/wcmc_components/loadable.rb:8]: I'm also going to remove the empty InstanceMethods module. I think this is probably from boilerplate or something like that, not having any effect on our code right now

Comment (Joseph-Burke) [reply wcmc_components/lib/wcmc_components/loadable.rb]: Addressed in ac6419d6d2aab90e84350aa2002cf27ba98502c9

File: lib/capistrano/tasks/config.rake
Comment (Joseph-Burke) [reply lib/capistrano/tasks/config.rake:73]: Hahaha that is weird.

Honestly I think it has no real effect on the code to have these interpolated strings here. I'm going to remove these so it looks more like how it was previously.

Screenshot 2022-10-27 at 10 58 32

Comment (Joseph-Burke) [reply lib/capistrano/tasks/config.rake]: Fixed in f5212904a3ce19cdadb6d71965fa4e126faf918b

**Comment by Joseph-Burke** [comment]
Merging these in without a second review

**Pull request merged**
